Backend/app/rider-platform/rider-app/Main/src/Tools/Maps.hs

```diff
@@ -105,11 +105,15 @@ getRoutes isAvoidToll personId merchantId mbMOCId req = do
   mOCId <- getMerchantOperatingCityId personId mbMOCId
   runWithServiceConfig (Maps.getRoutes $ fromMaybe merchant.isAvoidToll isAvoidToll) (.getRoutes) merchantId mOCId req
 
-getPickupRoutes :: ServiceFlow m r => Id Person -> Id Merchant -> Maybe (Id MerchantOperatingCity) -> GetRoutesReq -> m GetRoutesResp
-getPickupRoutes personId merchantId mbMOCId req = do
+getPickupRoutes :: ServiceFlow m r => Id Merchant -> Id MerchantOperatingCity -> MapsService -> GetRoutesReq -> m GetRoutesResp
+getPickupRoutes merchantId merchantOperatingCityId service req = do
   merchant <- SMerchant.findById merchantId >>= fromMaybeM (MerchantNotFound merchantId.getId)
-  mOCId <- getMerchantOperatingCityId personId mbMOCId
-  runWithServiceConfig (Maps.getRoutes merchant.isAvoidToll) (.getPickupRoutes) merchantId mOCId req
+  merchantMapsServiceConfig <-
```

Comment:
Can `runWithServiceConfig` function be used here with (.getPickupRoutes) or (.getFirstPickupRoute) changed accordingly ?

Reply:
We have set getFirstPickupRoute as Maybe Service so we can set it as NULL to disable query
So can't pass it